src/VisualStudio/IntegrationTest/New.IntegrationTests/CSharp/CSharpExtractMethod.cs

```diff
@@ -126,7 +126,7 @@ await TestServices.EditorVerifier.TextContainsAsync(@"private static int SayHell
     }", cancellationToken: HangMitigatingCancellationToken);
         }
 
-        [IdeFact]
+        [IdeFact(Skip = "https://github.com/dotnet/roslyn/issues/71088")]
```

Comment:
Note - we don't have to skip these now (they are passing on the current images).  But if we don't fix them by the time the queue upgrades, we will need to do this.

Reply:
Link to queue failure?

Reply:
@sharwell https://dev.azure.com/dnceng-public/public/_build?definitionId=98&_a=summary